lib/searchkick.rb

```diff
@@ -104,8 +104,14 @@ def self.search(term = "*", **options, &block)
   def self.multi_search(queries)
     if queries.any?
       responses = client.msearch(body: queries.flat_map { |q| [q.params.except(:body), q.body] })["responses"]
+
       queries.each_with_index do |query, i|
-        query.handle_response(responses[i])
+        if query.misspellings_below && responses[i]["hits"]["total"] < query.misspellings_below
+          query.send(:prepare)
+          query.execute
```

Comment:
Also, this is not great because it defeats the purpose of multi search. Any query that needs to be re-executed due to missing the misspelling results threshold will be run individually. 

If we can collect all the queries that need to be re-executed and run them all through `client.msearch` at once, that would be ideal.